index.html

```diff
@@ -5,8 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <h3 class="logo">Henry Cadena</h3>
+      <div class="container-burger">
+        <label class="burger1"></label>
+        <label class="burger2"></label>
+        <label class="burger3"></label>
+      </div>
+    </header>
+    <main class="container-main">
+      <img class="hero-image" src="assets/background-mobile-version.svg" alt="" />
```

Comment:
- [ ] To achieve a design similar to the suggested Figma design, it would be more appropriate to set the background image using CSS instead of a `<img>` tag. 
- [ ] By applying the background image through a CSS file, you have greater flexibility in manipulating the design and ensuring it matches the desired layout and style. 
- [ ] This approach allows for more control over positioning, sizing, and other visual aspects of the background image, resulting in a design that closely resembles the suggested Figma design.
- [ ] Your design:
![1](https://github.com/henrycadenaec/PortfolioMobileFirst/assets/118761303/8d819a3b-9610-4c54-9070-2a8124ad56c0)

- [ ] Figma design:
![figma](https://github.com/henrycadenaec/PortfolioMobileFirst/assets/118761303/98a34795-670c-4823-9427-fe2abfbd1659)